lesson1/index.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-
-    <title>TerraceTechフロントエンドエンジニア養成所</title>
-  </head>
-  <body>
-    <div>
-      <ul id="target"></ul>
-    </div>
-  </body>
-  <script src="index.js" async></script>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="style.css" type="text/css" />
+        <script src="index.js" async></script>
```

Comment:
Q
Why use async?

Reply:
Thank you for your comment.
I was using it for no particular reason.
I will make sure to use `defer` in the future.

Reply:
They may use async for advertising and access analysis.

[reference](https://penpen-dev.com/blog/script-async-defer/)

Reply:
I see!
I learned a lot.